pixeltable/env.py

```diff
@@ -1135,18 +1147,43 @@ def debug_str(self) -> str:
         )
 
     def update(self, recorded_at: datetime.datetime, limit: int, remaining: int, reset_at: datetime.datetime) -> None:
-        # we always update everything, even though responses may come back out-of-order: we can't use reset_at to
```

Comment:
Why remove that comment? Explain (in a comment) why this doesn't apply anymore/what assumptions you're making.